UnityProject/Assets/Scripts/Messages/Server/ServerMessage.cs

```diff
@@ -120,30 +120,17 @@ public static void SendToVisiblePlayers(Vector2 worldPosition, T msg, int channe
 
 		/// <summary>
 		/// Sends the network message only to players who are within a 15 tile radius
-		/// of the worldPostion. This method disregards if the player is visible or not
+		/// of the worldPosition. This method disregards if the player is visible or not
 		/// </summary>
 		public static void SendToNearbyPlayers(Vector2 worldPosition, T msg, int channel = 0)
 		{
-			var players = PlayerList.Instance.AllPlayers;
+			var players = PlayerList.Instance.InGamePlayers;
 
 			for (int i = players.Count - 1; i > 0; i--)
 			{
-				if (Vector2.Distance(worldPosition,
-					players[i].GameObject.transform.position) > 15f)
-				{
-					//Player in the list is too far away for this message, remove them:
-					players.Remove(players[i]);
-				}
-			}
-
-			foreach (ConnectedPlayer player in players)
-			{
-				if (player.Script == null) continue;
+				if (Vector2.Distance(worldPosition, players[i].GameObject.transform.position) > 15f) return;
```

Comment:
```suggestion
				if (Vector2.Distance(worldPosition, players[i].GameObject.transform.position) > 15f) continue;
```

Shouldn't this be continue?